src/seedu/addressbook/data/person/Address.java

```diff
@@ -36,24 +33,10 @@ public static boolean isValidAddress(String test) {
         return test.matches(ADDRESS_VALIDATION_REGEX);
     }
 
-    @Override
-    public String toString() {
-        return value;
-    }
-
     @Override
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof Address // instanceof handles nulls
                 && this.value.equals(((Address) other).value)); // state check
     }
-
-    @Override
-    public int hashCode() {
```

Comment:
if `equals()` is overriden, `hashCode()` also has to be overridden. you may find out why.